guides/source/users/promotions/promotion-calculators.html.md

```diff
@@ -0,0 +1,158 @@
+# Promotion calculators
+
+When you create a promotion action, one step is to select the promotion
+calculator that it should use. You can do this by selecting a value in the
+**Calculated By** drop-down menu.
+
+Depending on the [type of promotion action][promotion-action-types] that you are
+creating, only a subset of the promotion calculators may be available to use.
+
+This table outlines the outlines the available promotions calculators and which
+promotion action types they can be applied to:
```

Comment:
The header of this table is misleading. There are no promotion action types listed, these are calculables (a term we probably should explain as well).